ui/client/containers/Visualization.js

```diff
@@ -52,6 +52,9 @@ class Visualization extends React.Component {
     }
   }
 
+  allowBindCopyShortcut = () =>
+    this.props.allModalsClosed && _.isEmpty(this.props.selectionState) === false
```

Comment:
Wouldn't `!_.isEmpty(this.props.selectionState)` be more readable?

Reply:
Hmm, in future we want to rewrite this to TS, after that probably we will not use lodash ;)